src/app/child-dev-project/notes/dashboard-widgets/important-notes-dashboard/important-notes-dashboard.component.html

```diff
@@ -1,7 +1,11 @@
 <app-dashboard-list-widget
   icon="exclamation-triangle"
-  subtitle="Notes needing follow-up"
+  [subtitle]="subtitle ?? 'Notes needing follow-up'"
   i18n-subtitle="subtitle|dashboard showing notes that require action"
```

Comment:
sorry, I've rethought this to make the translation token easier to manage:

```suggestion
  [subtitle]="subtitle"
```

Move the translation to the component.ts, so that we do not have a condition ("??") within the translated string.

This requires in the component then:
```
  @Input() subtitle: string = $localize`:dashboard widget subtitle:Notes needing follow-up`;
```

@sadaf895 , could you please make such adjustment to the explanation property and the other components also? :blush: